source/gui/settingsDialogs.py

```diff
@@ -4714,13 +4733,19 @@ def makeSettings(self, settingsSizer):
 
 	def onSave(self):
 		AutoSettingsMixin.onSave(self)
-
-		braille.handler.table = self.outTables[self.outTableList.GetSelection()]
-		brailleInput.handler.table = self.inTables[self.inTableList.GetSelection()]
+		if self.outTableList.GetSelection():
```

Comment:
Please make this more explicit:
```suggestion
		if self.outTableList.GetSelection() > 0:
```